packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec_overrides.yaml

```diff
@@ -1,12 +1 @@
 dependency_overrides:
-  ## TODO: Temporary fix until [this issue](https://github.com/jhomlala/alice/issues/234) is closed
```

Comment:
Do we still need this file since it's now empty ?

Reply:
No, Removed